recipes/openvino-tokenizers/meta.yaml

```diff
@@ -0,0 +1,116 @@
+{% set name = "openvino-tokenizers" %}
+{% set version = "2023.2.0" %}
+
+{% set install_args = "-vv --config-settings=wheel.cmake=False" %}
+
+# Libraries on Windows are expected to be installed into LIBRARY_PREFIX.
+# See https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html.
+# On non-Windows CMAKE_ARGS already contains CMAKE_INSTALL_PREFIX.
+{% if win %}
+    {% set cmake_args = "%CMAKE_ARGS% -DCMAKE_INSTALL_PREFIX=%LIBRARY_PREFIX%" %}
+{% else %}
+    {% set cmake_args = "$CMAKE_ARGS" %}
+{% endif %}
+
+package:
+  name: {{ name|lower }}-split
+  version: {{ version }}
+
+source:
+  url: https://github.com/openvinotoolkit/openvino_contrib/archive/4c60cb1b20f45baef27b4e79fba30f929359987b.tar.gz
```

Comment:
it's an initial release of this component against latest OpenVINO 2023.2
Later, OpenVINO tokenizers will be aligned with OpenVINO and have coupled releases.